core/src/main/scala/scalaz/StateT.scala

```diff
@@ -73,26 +74,26 @@ sealed abstract class IndexedStateT[F[_], -S1, S2, A] { self =>
     )
 
   import Liskov._
-  def unlift[M[_], FF[_], S <: S1](implicit M: Comonad[M], F: Monad[λ[α => M[FF[α]]]], ev: this.type <~< IndexedStateT[λ[α => M[FF[α]]], S, S2, A]): IndexedStateT[FF, S, S2, A] = IndexedStateT.createState(
+  def unlift[M[_], FF[_], S](implicit M: Comonad[M], F: Monad[λ[α => M[FF[α]]]], ev: this.type <~< IndexedStateT[λ[α => M[FF[α]]], S, S2, A]): IndexedStateT[FF, S, S2, A] = IndexedStateT.createState(
```

Comment:
The S1 subtype relationship is no longer here (its gone on `unliftId` as well.)  I think this is fune, because of the subtype relationship on IndexedStateT. Just thought I'd mention it in case this may cause an issue I'm unaware of.